stl/inc/bit

```diff
@@ -12,6 +12,7 @@
 #pragma message("The contents of <bit> are available only with C++20 or later.")
 #else // ^^^ !_HAS_CXX20 / _HAS_CXX20 vvv
 
+#include <intrin.h>
 #include <limits>
 #include <type_traits>
 

```

Comment:
You're now defining `__cpp_lib_bitops` unconditionally (in C++20 mode), so the `#ifdef __cpp_lib_bitops // TRANSITION, VSO-1020212` below should be removed. Please audit the whole STL and its tests for now-unnecessary checks of both `__cpp_lib_bitops` and `__cpp_lib_int_pow2`.